tests/console/shibboleth.pm

```diff
@@ -21,7 +21,7 @@ sub run {
 
     zypper_call "in shibboleth-sp apache2";
     assert_script_run "a2enmod shib";
-    assert_script_run "systemctl restart apache2.service";
+    systemctl 'restart apache2';
```

Comment:
Sometime we have ".service", sometimes we don't. Let's make it consistent as you start changing it.

Reply:
I don't think we have to. If it's ambiguous, e.g. there is more than just a service for one name then we should add the service suffix.